packages/utils/telemetry-utils/src/sampledTelemetryHelper.ts

```diff
@@ -0,0 +1,111 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    IDisposable,
+    ITelemetryGenericEvent,
+    ITelemetryLogger,
+    ITelemetryPerformanceEvent,
+} from "@fluidframework/common-definitions";
+import { performance } from "@fluidframework/common-utils";
+
+interface Measurements {
+    // The names of the properties in this interface are the ones that will get stamped in the
+    // telemetry event, changes should be considered carefully. The optional properties should
+    // only be populated if 'includeAggregateMetrics' is true.
+
+    duration: number;
+    count?: number;
+    totalDuration?: number;
+    minDuration?: number;
+    maxDuration?: number;
+}
+
+interface CountAndMeasurements { count: number; measurements: Measurements; }
```

Comment:
Because then a `count` property would show up in the telemetry event regardless of the value of `includeAggregatedMetrics`, and I thought it would be confusing for it to be there if the consumer only wants the data for the sample itself. I wasn't thrilled with the apparent duplication that resulted... and just now an alternative came to mind, assuming TS will allow it if `count` is required: right before writing the event, check `includeAggregatedMetrics` and if `false`, remove `count` from the property bag. How does that sound to both of you? (cc @anthony-murphy).

Reply:
i like count even  in the sampling case to understand our sample rate

Reply:
so i would just always have it